src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -0,0 +1,256 @@
+                                               # while `uEltype` is used as element type of solutions etc.
+                                               RealT=real(solver), uEltype=RealT,
+                                               initial_caches=(NamedTuple(), NamedTuple()))
+
+  hyperbolic_equations, equations_parabolic = equations
+  hyperbolic_boundary_conditions, parabolic_boundary_conditions = boundary_conditions
+  initial_hyperbolic_cache, initial_parabolic_cache = initial_caches
+
+  return SemidiscretizationHyperbolicParabolic(mesh, hyperbolic_equations, equations_parabolic,
+                                               initial_condition, solver; source_terms,
+                                               boundary_conditions=hyperbolic_boundary_conditions,
+                                               parabolic_boundary_conditions=parabolic_boundary_conditions,
+                                               RealT, uEltype, initial_cache=initial_hyperbolic_cache,
+                                               initial_parabolic_cache=initial_parabolic_cache)
+end
+
+function SemidiscretizationHyperbolicParabolic(mesh, equations, equations_parabolic,
+                                               initial_condition, solver;
+                                               source_terms=nothing,
+                                               boundary_conditions=boundary_condition_periodic,
+                                               parabolic_boundary_conditions=boundary_condition_periodic,
+                                               # `RealT` is used as real type for node locations etc.
+                                               # while `uEltype` is used as element type of solutions etc.
+                                               RealT=real(solver), uEltype=RealT,
+                                               initial_cache=NamedTuple(),
+                                               initial_parabolic_cache=NamedTuple())
+
+  cache = (; create_cache(mesh, equations, solver, RealT, uEltype)..., initial_cache...)
+  _boundary_conditions = digest_boundary_conditions(boundary_conditions, mesh, solver, cache)
+  _parabolic_boundary_conditions = digest_boundary_conditions(parabolic_boundary_conditions, mesh, solver, cache)
+
+  parabolic_cache = (; create_cache(mesh, equations_parabolic, solver, RealT, uEltype)...,
+                       initial_parabolic_cache...)
+
+  SemidiscretizationHyperbolicParabolic{typeof(mesh), typeof(equations), typeof(equations_parabolic),
+                                        typeof(initial_condition), typeof(_boundary_conditions), typeof(_parabolic_boundary_conditions),
+                                        typeof(source_terms), typeof(solver), typeof(cache), typeof(parabolic_cache)}(
+    mesh, equations, equations_parabolic, initial_condition,
+    _boundary_conditions, _parabolic_boundary_conditions, source_terms,
```

Comment:
```suggestion
    _boundary_conditions, _boundary_conditions_parabolic, source_terms,
```
Here and (potentially) elsewhere in the file.